src/WattleScript.Interpreter/Tree/Expression_.cs

```diff
@@ -321,6 +321,8 @@ private static Expression PrefixExp(ScriptLoadingContext lcontext)
 			Token T = lcontext.Lexer.Current;
 			switch (T.Type)
 			{
+				//case TokenType.Op_InclusiveRange when lcontext.Syntax == ScriptSyntax.WattleScript:
```

Comment:
Accidental commit

Reply:
fixed in [9d78d89](https://github.com/WattleScript/wattlescript/pull/36/commits/9d78d899a2c1edb6f9650297c230cdabdf2bcb7f)